src/gobupload/update/event_applicator.py

```diff
@@ -99,11 +99,7 @@ def apply(self, event):
         count = 1
         tid = gob_event.tid
 
-        if isinstance(gob_event, GOB.BULKCONFIRM):
```

Comment:
This path is not used anymore. Confirm events are not stored in the events table